docs/wiki/Getting started - Scenario 2 Onboard module library and CI environment.md

```diff
@@ -90,26 +96,28 @@ To lower the barrier to entry and allow users to easily define their own naming
 
 > **Note:** This prefix is only used by the CI environment you validate your modules in, and doesn't affect the naming of any resources you deploy as part of any multi-module solutions (applications/workloads) based on the modules.
 
-Each pipeline in CARML deploying resources uses a logic that automatically replaces "tokens" (i.e., placeholders) in any parameter file. Tokens are stored in only a few central locations to facilitate maintenance (e.g., local `settings.json`, repository secrets or variable groups).
+Each pipeline in CARML deploying resources uses a logic that automatically replaces "tokens" (i.e., placeholders) in any parameter file. Tokens are stored in only a few central locations to facilitate maintenance (e.g., local `settings.yml`, repository secrets or variable groups).
 
 To update the `namePrefix`, perform the following steps:
 
-1. Open the `settings.json` file in the repository's root directory.
+1. Open the `settings.yml` file in the repository's root directory.
 
-1. Replace the `"value": "<...>"` of token `namePrefix` with a different value:
+1. Replace the value of the `localToken_namePrefix` with a different value:
 
-    ```json
-    {
-        "name": "namePrefix",
-        "value": "<...>"
-    }
+    ```yml
+    localToken_namePrefix: 'cntso'
     ```
+
     > **Note:** The value should be a 3-5 character long string like `cntso`. Longer strings are not recommended as they may conflict with Azure resource name length restrictions.
 
+    > **Note:** The CI pipelines automatically removes `localToken_` section from the name when processing the tokens replacement.
+
     > **Note:** We highly encourage you to use the 'Check namePrefix availability' script ([see the documentation here](./Getting%20started%20-%20Check%20NamePrefix%20availability)) to check if the intended resource name will be available, based on the provided prefix.
 
  For further information on the token replacement logic, please refer to the corresponding [Token replacement](./The%20CI%20environment%20-%20Token%20replacement) section.
 
+ If you do not prefer to set the `namePrefix` locally for your CI environment. You can optionally set it as a GitHub Secret or add it to the ADO variable group as the key `TOKEN_NAMEPREFIX` with value `your name prefix value` and that will take precedence over the one specified in the `settings.yml`.
```

Comment:
I'd be more opinionated in suggesting to use the github secret over the local one. This is especially for the contribution scenario since contributors won't need to change the nameprefix value as a last thing anymore (finally ❤️ )

Reply:
For contributors yes, but for anybody onboarding it in a customer scenario, I assume they'd go with the settings file as it is more transparent and descriptive.

Reply:
While I appreciate the possibility to set the prefix in the secrets, to me it feels more like a change for us, rather than anybody else. The namePrefix is after all technically not a secret ^^

Reply:
Although tested multiple times across ADO and GHA, I have left the current local name prefix to be carml. Do you reckon we should convert it to something like `cntso` ? As we want to obscure our token to protect our CI env name?